docs/index.rst

```diff
@@ -1,16 +1,21 @@
-.. td-client-python documentation master file, created by
-   sphinx-quickstart on Fri Sep  6 10:12:53 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+td-client - Treasure Data API library for Python
+========================================================
+
+``td-client`` is a Python client for Arm Treasure Data's REST API.
```

Comment:
Do we really need this (and following) line? The generated index page repeatedly explains the same statement...
<img width="718" alt="Screen Shot 2019-09-27 at 9 41 56 AM" src="https://user-images.githubusercontent.com/853567/65733807-200b1d80-e10b-11e9-84f3-d925d551d06f.png">

Reply:
fixed f994fd5